### PR TITLE
Allow felix build on 32 bits systems (armv7)

### PR DESCRIPTION
**Pull request by aalaesar**
## Description
A small patch allowing to build felix for 32 bits architecture 
- fix : change tc.MarksMask datatype to unsigned 32 bits integer. Somehow this 32 bits value is picked as 64 bits integer.
- required to build felix on armv7
- tested locally by cross compiling calico/node  for armv7 or amd64
- component affected: calico/felix & calico/node

## Todos
- [ ] Unit tests
- [ ] Integration tests:  Not needed ?

## Release Note

```release-note
None required
```

### CODE REVIEW

**Review by fasaxc — APPROVED** [review]
LGTM

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
